sketch/README.md

```diff
@@ -0,0 +1,197 @@
+# Wifi Localization sketch
+
+The sketch is built upon the use of the following hardware and components:
+
+* Sodaq ExpLoRer
+* ESP8266-01
+* [*FTDI USB to TTL logic*](https://www.amazon.com/HiLetgo-FT232RL-Converter-Adapter-Breakout/dp/B00IJXZQ7C/ref=pd_lpo_vtph_147_lp_img_3/131-9012372-9811114?_encoding=UTF8&psc=1&refRID=PH0S6PBC54E7YFZYEKZD)
+* Breadboard
+* 5 male-female, 4 female-female wires and 1 male-male wire.
+
+### Index
+---
+* Hookup guide
+* Important values/calculations
+* Flow of the program
+* Functions
+
+### Hookup guide
+---
+##### **Connecting the ExpLoRer**
+
+The Sodaq ExpLoRer is a development board that has a potential of 3.3 volts on all pins except (ofcourse) the 5 volt pin. So it is possible to connect the esp directly to the Sodaq board. The wiring can be seen in the image down below. 
```

Comment:
- remove `of course`, don't assume people are smart
- `esp` -> `ESP`. Be consistent with naming always everywhere

Reply:
Yeah i was already doubting the of course, will remove it. but it makes sense that a pin named the 5 volt pin isn't 3.3 volt right.